eitprocessing/binreader/sequence.py

```diff
@@ -199,14 +296,17 @@ def read(self, limit_frames: slice = None, framerate: int = 20):
 
             reader = Reader(fh)
             for index in range(self.n_frames):
-                self.read_frame(reader, index, pixel_values)
+                self.read_frame_draeger(reader, index, pixel_values)
```

Comment:
the function name below was not updated to match this.

Reply:
I have fixed this in [f003360](https://github.com/EIT-ALIVE/eitprocessing/pull/68/commits/f00336018a7691850e4746619c48f3efceeb2f72).